mwdb/web/src/components/Upload/common/UploadDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+import { faUpload } from "@fortawesome/free-solid-svg-icons";
+import { NavDropdown } from "@mwdb-web/commons/ui";
+import { Capability } from "@mwdb-web/types/types";
+import { UploadButton } from "../../UploadButton";
+
+export function UploadDropdown() {
+    const uploadElement = [
```

Comment:
I would hide additional dropdown elements when user doesn't have capability to upload configs or blobs. If no upload is possible, button should be hidden. If only one type is possible, I would make it single button described as `Upload` (for files), `Upload blob` (for blobs), `Upload config` (for configs)